### PR TITLE
HTML-705: Condition tag support from Core 2.2.0 and beyond.

### PR DESCRIPTION
**Pull request by samuelmale**
Followup commit on => https://issues.openmrs.org/browse/HTML-705

### CODE REVIEW

**Comment by samuelmale** [comment]
Yes, it works as expected @mks-d

File: api-2.2/src/main/java/org/openmrs/module/htmlformentry/handler/ConditionTagHandlerSupport2_2.java [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import org.openmrs.module.htmlformentry.FormEntrySession;
 import org.openmrs.module.htmlformentry.FormSubmissionController;
 
-@OpenmrsProfile(openmrsPlatformVersion = "2.2.*")
+@OpenmrsProfile(openmrsPlatformVersion = "2.2.0")
```
Comment (mogoodrich) [inline api-2.2/src/main/java/org/openmrs/module/htmlformentry/handler/ConditionTagHandlerSupport2_2.java]: @samuelmale  @mks-d  what is the purpose behind this change?
